planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
they come out from the AAM and are needed for adding the monitoring rules to the DAM. After that `NodeTypes` are no more needed because @kiuby88 included it in the Deployer.